frontend/packages/ux-editor/src/containers/DesignView/PageAccordion/NavigationMenu/InputPopover/InputPopover.tsx

```diff
@@ -134,6 +131,15 @@ export const InputPopover = ({
           {errorMessage}
         </ErrorMessage>
         <div className={classes.buttonContainer}>
+          <Button
+            color='first'
+            variant='filled'
+            onClick={() => errorMessage === null && newName !== oldName && saveNewName(newName)}
+            aria-disabled={errorMessage !== null || newName === oldName}
```

Comment:
Her kommer den samme sjekken to ganger, bare at den ene er invers av den andre. Jeg ville satt disse inn i en variabel med navnet `shoouldSavingBeEnabled` e.l. Da er det også mer lesbart.

Er det for øvrig noen grunn til at du bruker `aria-disabled` og ikke `disabled`? Hvis du bruker `disabled`, blir jo ikke `onClick` kalt uansett, og da trenger du ikke å gjøre den samme sjekken der.

Reply:
Ja enig, fikser det nå :)